.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Before opening you Pull Request have a quick look at our Contributing Guidelines.
+
+When adding a new or updating an icon remember the following points:
+* Include the icon’s brand name in the title.
+* Tell us why you chose the color you picked.
+* Make sure you optimized the icon with SVGO or SVGOMG.
+* Reference the issue, if any, the Pull Request addresses.
```

Comment:
Perhaps add an "make sure the icon has been resized to have a viewbox of `0 0 24 24`", as that's a problem that keeps popping up